transforms/no-implicit-this/index.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 const fs = require('fs');
 
+const debug = require('debug')('ember-no-implicit-this-codemod:transform');
```

Comment:
Does this string have to match to value of the env var?

Reply:
yes, but the env var does support wildcards